packages/date-picker/src/CalendarNav.tsx

```diff
@@ -0,0 +1,32 @@
+import React from "react"
+import { Icon } from "@kaizen/component-library"
+import arrowRight from "@kaizen/component-library/icons/arrow-right.icon.svg"
+import arrowLeft from "@kaizen/component-library/icons/arrow-left.icon.svg"
+import styles from "./DatePicker.scss"
+
+export type CalendarNavProps = {
+  onPreviousClick: () => void
+  onNextClick: () => void
+}
+
+export const CalendarNav: React.FunctionComponent<CalendarNavProps> = ({
```

Comment:
Just wondering, is `React.FunctionComponent` the one that has a `children` prop baked in - which we wouldn't want here, would we? I think @HeartSquared always uses a different one that I can't remember.

Reply:
This does have `PropsWithChildren` baked in it seems. Keen to know if there's one we prefer, I always reach for this.

Reply:
If we don't expect `children`, then I'd say we shouldn't have them in the type declaration. Not that it really could do much harm if someone did
```
<DatePicker><h1>Hello World</h1></DatePicker>
```
but for cleanliness' sake.

Btw, I think it's `React.VoidFunctionComponent` that could be used, but I'd also ask @HeartSquared as I know she has some views on this.

Reply:
Yes `VoidFunctionComponent` aka `VFC` is the one without children. There was an article somewhere which mentioned that in a future major version they plan to remove children from `FC` and then deprecate `VFC`, but it hasn't been solidified yet, so in the meantime `VFC` is recommended as it is more type safe. If you do need `children`, then you would explicitly add it to your props since it adds more clarity and gives you full control of the type signature.

It's definitely on the list for Wednesday's meeting :)

Reply:
Thanks for that, it's now been updated.

Reply:
Here's a similar article to what I had read previously :) 
https://spin.atomicobject.com/2022/01/04/think-twice-react-fc/